test/cri-containerd/jobcontainer_test.go

```diff
@@ -326,3 +327,79 @@ func Test_RunContainer_HostVolumes_JobContainer_WCOW(t *testing.T) {
 		t.Fatalf("expected volumes to be the same within job process container. got %q but expected %q", hostStdout, containerStdout)
 	}
 }
+
+func Test_RunContainer_LocalService(t *testing.T) {
```

Comment:
this test isn't related to the changes in this PR is it?

Reply:
No

Reply:
Can we move it to a different PR then?

Reply:
Sure